app/user.go

```diff
@@ -4,13 +4,16 @@ import "github.com/ubclaunchpad/cumulus/blockchain"
 
 // User holds basic user information.
 type User struct {
-	// Wallet holds the users wallets (currently just support 1).
-	Wallet blockchain.Wallet
-	// UserBlockSize is the maximum size of a block in bytes when marshaled
-	// as specified by the user.
+	HotWallet
 	BlockSize uint32
 }
 
+// HotWallet is a representation of the users wallet.
+type HotWallet struct {
```

Comment:
Practically speaking, it's in memory.

https://en.bitcoin.it/wiki/Hot_wallet

At the moment it's a wrapper to get a name on it the wallet.